src/main.js

```diff
@@ -473,6 +473,8 @@ class WeatherChartCard extends LitElement {
     }
     if (this.unitPressure === 'mmHg') {
       pressure = Math.round(pressure * 0.75);
+    } else if (this.unitPressure === 'in' ) {
```

Comment:
Should it be 'inHg'?

Reply:
Yeah, it should, but the way units are tied to language then you get no units.  Maybe this PR should be closed and instead, https://github.com/Yevgenium/weather-chart-card/issues/82 would be a good discussion on the right way to resolve this as language and units probably should be separated.  In the US, our weather APIs return pressure in inHg, so no calculations needed (per your other comment), so this was my compromise to improve the display while figuring out the preferred way to separate units and language.